src/main/java/org/sigmah/server/servlet/exporter/template/IndicatorEntryCalcTemplate.java

```diff
@@ -211,16 +213,76 @@ private void createDetailSheet(final IndicatorDTO indicator) throws Throwable {
 		row.getCellByIndex(4).setCellStyleName(null);
 
 		// data entry
+        //for year row in sheet
+        String merge = "hello";
+        row = tableEx.getRowByIndex(++rowIndex);
+        int cellIndex = 0;
+        CalcUtils.putHeader(row, ++cellIndex, "Year");
+        Map<String, Integer> columnIndexMapyr = new HashMap<String, Integer>();
+        for (PivotTableData.Axis axis : leavesyear) {
+            CalcUtils.putHeader(row, ++cellIndex, axis.getLabel());
+            columnIndexMapyr.put(axis.getLabel(), cellIndex);
+            if (merge.equals(axis.getLabel()) && cellIndex > 2) {
+                CalcUtils.mergeCell(tableEx, cellIndex - 1, rowIndex, cellIndex, rowIndex);
+            } else {
+            }
+            merge = axis.getLabel();
+
+        }
+        int month = 0;
+        String completedate = "hello";
```

Comment:
Please, follow java conventions and use ``lowerCamelCase`` for variable names (``completedate``, ``leavesyear`` lacks uppercases)